easybuild/easyconfigs/l/LAMMPS/LAMMPS-3Mar2020-intel-2020a-Python-3.8.2-kokkos.eb

```diff
@@ -0,0 +1,107 @@
+name = 'LAMMPS'
+version = '3Mar2020'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
+for Large-scale Atomic/Molecular Massively Parallel Simulator. LAMMPS has
+potentials for solid-state materials (metals, semiconductors) and soft matter
+(biomolecules, polymers) and coarse-grained or mesoscopic systems. It can be
+used to model atoms or, more generically, as a parallel particle simulator at
+the atomic, meso, or continuum scale. LAMMPS runs on single processors or in
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+# 'https://github.com/lammps/lammps/archive/'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = [
+    'stable_%(version)s.tar.gz',
+    {'filename': 'lammps_vs_yaff_test_single_point_energy.py', 'extract_cmd': "cp %s %(builddir)s"},
+]
+patches = ['LAMMPS-3Mar2020_fix-docs-build.patch']
+checksums = [
+    'a1a2e3e763ef5baecea258732518d75775639db26e60af1634ab385ed89224d1',  # stable_3Mar2020.tar.gz
+    '5e5211f64113e8a38c102f8542fd59169b689fdb92afbff29c3b88f802f1a96c',  # lammps_vs_yaff_test_single_point_energy.py
+    '7f010853d81022f286cf32e3babe252d5cc7c0bfb274bee5a2c64e810e170239',  # LAMMPS-3Mar2020_fix-docs-build.patch
+]
+
+local_source_dir_name = '%(namelower)s-%(version)s'
+
+builddependencies = [
+    ('CMake', '3.16.4'),
+    ('pkg-config', '0.29.2'),
+    ('archspec', '0.1.0', local_python_versionsuffix),
+]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.4'),
+    ('netCDF', '4.7.4'),
+    ('GSL', '2.6'),
+    ('zlib', '1.2.11'),
+    ('gzip', '1.10'),
+    ('cURL', '7.69.1'),
+    ('HDF5', '1.10.6'),
+    ('tbb', '2020.1'),
+    ('PCRE', '8.44'),
+    ('libxml2', '2.9.10'),
+    ('FFmpeg', '4.2.2'),
+    ('Voro++', '0.4.6'),
+    ('kim-api', '2.1.3'),
+    ('Eigen', '3.3.7'),
+    ('yaff', '1.6.0', local_python_versionsuffix),
+    ('PLUMED', '2.6.0', local_python_versionsuffix),
+    ('ScaFaCoS', '1.0.1'),
+    # See below for why this is not included
+    # ('VTK', '8.2.0', local_python_versionsuffix),
+]
+
```

Comment:
```suggestion

# Build the documentation with an initialized MPI, otherwise sphinx-build will crash.
# This is known issue with intel/2020a. See easybuilders/easybuild-easyconfigs issue #10213
local_doccmake = '%(builddir)s/lammps-stable_%(version)s/cmake/Modules/Documentation.cmake'
preconfigopts = "sed -i 's/sphinx-build/%%(mpi_cmd_prefix)s sphinx-build/g' %s &&" % local_doccmake

```

Reply:
This cannot be patched because it depends on `%(mpi_cmd_prefix)s`